docs/cloud-soar/cloud-soar-apis.md

```diff
@@ -8,27 +8,27 @@ description: Learn how to access Cloud SOAR APIs and API documentation.
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import ApiIntro from '../reuse/api-intro.md';
 
-<!-- Add this once all customers are moved to Cloud SOAR Delivery 2, and remove all the documentation below for "incmansuite_ng:
-
 The Cloud SOAR APIs allow you to manage incidents, triage, and other Cloud SOAR features.
 
-<ApiIntro/>
-
-import CloudSoarApi from '../reuse/csoar-api-table.md';
+:::info
+This article applies to your organization *only if* your instance URL is a subdomain of `soar.sumologic.com`.
+Otherwise, please refer to:
 
-<CloudSoarApi/>
+* [Cloud SOAR APIs](/docs/api/cloud-soar) for the endpoints by geographic location
+* [API Authentication](/docs/api/getting-started#authentication) for details about API best practises.
 
--->
+:::
 
-### API documentation
+### Inline API documentation
 
-API documentation is available through your Cloud SOAR instance at the following URL:
+For organizations having Cloud SOAR available at URL pattern `*.soar.sumologic.com`,
+the API documentation is available through your Cloud SOAR instance at the following URL:
```

Comment:
```suggestion
For organizations with a Cloud SOAR API URL pattern of `*.soar.sumologic.com`, you can find API documentation is through your Cloud SOAR instance at:
```

Reply:
Thanks for the review.
I rewrote the sentences to be similar and to better clarify the situation.
To add more context:
We have customers with "legacy" Cloud SOAR instances that have been provisioned with URLs having a subdomain of "soar.sumologic.com". These customers could have standard Sumo Logic instances with regional and custom alias or could not have any Sumo Logic instance at all. So the rule to identify which part of the document applies to "Delivery 1" instances is to check if the Cloud SOAR application has a URL in the browser URL bar matching the "*.soar.sumologic.com" pattern (despite the their actual Sumo Logic organization URL). 
So the URL to check is not only the API one but any URL the customers use to reach Cloud SOAR (from browser too).

Hope that the new version of the text is better from your point of view.
Thanks